packages/dai-plugin-mcd/src/EventHistory.js

```diff
@@ -157,7 +163,13 @@ export default async function getEventHistory(cdpManager, managedCdp, cache) {
 
           const [joinDaiEvents, cdpMoveEvents] = await Promise.all([
             web3.getPastLogs({
-              address: [MCD_JOIN_DAI, MCD_JOIN_SAI],
+              /*
+              FIXME: ethers 5 doesn't support passing an array of addresses into getLogs
+              as we move further away from SCD, join Sai events are less likely, but
+              we should eventually re-add this functionality by duplicating this call for MCD_JOIN_SAI.
+              */
+              // address: [MCD_JOIN_DAI, MCD_JOIN_SAI],
+              address: MCD_JOIN_DAI,
```

Comment:
Same question here. How do you think we should address this?

Reply:
I left it there commented out as a reminder, but I don't know if we should prioritize it at the moment. Given that nobody uses the event history in the mcd plugin anymore since Oasis moved away from it, and given that it only affects parsing SCD events, I'm willing to release this version without it. If anyone feels differently let me know.

The fix would be to duplicate the same function using `MCD_JOIN_SAI` as the address, merge & sort the results.